### ISSUE TITLE
In-game menu bug

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#124]
```
When I start a new game the menu popping up, I click back and after ~2 seconds 
the menu is popping up again and again... 

Exception in function show at Sun Jul 17 09:16:44 2011 CEST:
Traceback (most recent call last):
  File "/root/Desktop/endgame-singularity-read-only/code/safety.py", line 58, in safe_call
    return func(*args, **kwargs)
  File "/root/Desktop/endgame-singularity-read-only/code/graphics/dialog.py", line 197, in show
    Dialog.top.maybe_update()
  File "/root/Desktop/endgame-singularity-read-only/code/graphics/widget.py", line 303, in maybe_update
    self.update()
  File "/root/Desktop/endgame-singularity-read-only/code/graphics/widget.py", line 307, in update
    self.prepare_for_redraw()
  File "/root/Desktop/endgame-singularity-read-only/code/graphics/widget.py", line 299, in prepare_for_redraw
    child.prepare_for_redraw()
  File "/root/Desktop/endgame-singularity-read-only/code/graphics/widget.py", line 286, in prepare_for_redraw
    self.rebuild()
  File "/root/Desktop/endgame-singularity-read-only/code/screens/map.py", line 523, in rebuild
    (g.to_money(g.pl.cash), g.to_money(g.pl.future_cash()))
  File "/root/Desktop/endgame-singularity-read-only/code/g.py", line 318, in to_money
    to_return = add_commas(amount)
  File "/root/Desktop/endgame-singularity-read-only/code/g.py", line 253, in add_commas
    grouping=True    ).decode(encoding)
TypeError: decode() argument 1 must be string, not None
```

Original issue reported on code.google.com by `dwlg...@web.de` on 17 Jul 2011 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
I'm using Python 2.7.1 under Arch Linux, kernel 2.6.39.3 64bit
```

Original comment by `dwlg...@web.de` on 17 Jul 2011 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
It happens at every update(redraw)
```

Original comment by `dwlg...@web.de` on 17 Jul 2011 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
Could you please confirm if the problem still exist in latest git commit? This 
issue looks related to missing translations, and a lot of the translating code 
was re-written. Also, does it happen in all languages?
```

Original comment by `singular...@rodrigosilva.com` on 15 May 2012 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Not sure if this is necro-posting or not... if so, i can create a new issue. 
Escape menu continues to pop up no matter what I do in arch linux. Unable to 
test on different distro at the moment.

uname -a
Linux cocytus 3.4.4-2-ARCH #1 SMP PREEMPT Sun Jun 24 18:59:47 CEST 2012 x86_64 
GNU/Linux

Issue persists whether I install and run package from the arch linux repository 
or I download the latest git version via:
git clone https://code.google.com/p/endgame-singularity/
and run via:
python2.7 singularity.py

I am running pygame 1.9.1-5

I would be more than happy to test in other languages, if explained how.

Edit:
I'm using xmonad 0.10-4 as my window manager
```

Original comment by `willi...@gmail.com` on 22 Jul 2012 at 9:37

**Comment by GoogleCodeExporter** [comment]
```
If the problem still persists, necro-posting is not an issue.

Could you please follow these steps so I can ivestigate it?
- Delete your settings folder (look for both ~/.endgame and 
~/.config/singularity)
- Make sure you're running the latest version for the master branch in git 
repository (commit b8d25ed426d32548246d5ad1a15eda280d51d69c, "update maketree 
so it works again")
- Run the game and try to reproduce the issue
- Zip and attach these files:
  - ~/.config/singularity/error.log and prefs.dat
  - If you are able to save the game while in this condition, include the saved game
  - The full terminal output, including prompt and command issued to run
  - the output of "locale" command

With those in hand, I hope I can nail the bug ;)

Last but not least: why are you running the game as root? Does it work when 
clonned and run as a regular user?
```

Original comment by `rodrigo....@gmail.com` on 16 Nov 2012 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git commit 6ece1e1c0aa0eef5c5a3be33abacafe6b9b8050c , can you please 
verify?
```

Original comment by `singular...@rodrigosilva.com` on 28 Nov 2012 at 10:40
* Changed state: **Fixed**
